content/well-architected-framework/docs/docs/define-and-automate-processes/process-automation/fully-automated.mdx

```diff
@@ -44,17 +46,32 @@ You can use Terraform and Packer to install and deploy monitoring agents into yo
 
+## Blue-green and canary deployments
+
+Blue/green, canary, and rolling deployments all improve application reliability and reduce risk. While they share similar goals, each approach offers unique advantages that make it more suitable for certain types of applications or organizational needs. By choosing the most appropriate deployment method, companies can ensure smoother updates and reduce the likelihood of service disruptions.
+
+- Blue/green deployments maintain two identical production environments concurrently. This method allows you to shift traffic from the current version (blue) to the upgraded version (green).
+- Canary deployments introduce new versions incrementally to a subset of users. This approach lets you test upgrades with limited exposure, working alongside other deployment systems.
+- Rolling deployments update applications gradually across multiple servers. This technique ensures only a portion of your infrastructure changes at once, reducing the risk of widespread issues.
+
+You can learn more about these deployment methods in the [Zero-downtime deployments](/well-architected-framework/define-and-automate-processes/deploy/zero-downtime-deployments) document set.
+
 HashiCorp resources:
 - Learn how to use [VCS-driven workflow](/terraform/tutorials/cloud-get-started/cloud-vcs-change) with HCP Terraform
 - Learn how to [automate Packer with GitHub Actions](/packer/tutorials/cloud-production/github-actions)
-- [What is HCP Packer?](/hcp/docs/packer)
-- [What is HCP Waypoint?](/hcp/docs/waypoint)
-- [Get started with HCP Waypoint](/hcp/docs/waypoint)
 - [Identify common metrics](/well-architected-framework/optimize-systems/monitor-system-health/identify-metrics)
+- Learn how to setup [monitoring agents](/well-architected-framework/define-and-automate-processes/monitor/setup-monitoring-agents)
+- Learn how to setup [dashboards and alerts](/well-architected-framework/define-and-automate-processes/monitor/setup-dashboards-and-alerts)
 
 External resources:
 - [What is GitOps?](https://about.gitlab.com/topics/gitops/)
 
 ## Next steps
 
-In this section of Process automation, you learned best practices for implementing fully-automated deployments. Implement fully-automated deployments is part of the [Define and automate processes pillar](/well-architected-framework/define-and-automate-processes).
\ No newline at end of file
+In this section of Process automation, you learned best practices for implementing fully-automated deployments. Implement fully-automated deployments is part of the [Define and automate processes pillar](/well-architected-framework/define-and-automate-processes).
+
+You can learn to get with Packer and Waypoint in the following documents:
+
+- [What is HCP Packer?](/hcp/docs/packer)
+- [What is HCP Waypoint?](/hcp/docs/waypoint)
+- [Get started with HCP Waypoint](/hcp/docs/waypoint)
```

Comment:
i think packer, terraform and nomad are the links here. waypoint doesn't really help with deployments iirc